setup/FSharp.Setup.props

```diff
@@ -11,7 +11,7 @@
     </PropertyGroup>
 
     <PropertyGroup>
-        <FSharpProductVersion>10.1</FSharpProductVersion>
+        <FSharpProductVersion>15.7</FSharpProductVersion>
```

Comment:
I don't see `$(FSharpProductVersion)` referenced anywhere?

Reply:
It's used in `setp\fsharp-setup-build.proj` and passed through to `setup\Swix\Microsoft.FSharp.Dependencies\Microsoft.FSharp.Dependencies.swixproj`.

Reply:
That appears to be passing FSharpPackageVersion, not FSharpProductVersion -
 https://github.com/Microsoft/visualfsharp/blob/master/setup/fsharp-setup-build.proj#L48

Reply:
Sorry, I missed a step.  3 lines below this change `<FSharpPackageVersion>` is set using `$(FSharpProductVersion)` as its prefix.

Reply:
Oh, I missed that too.  Just to be complete - where are the other versions compiler, language, FSharp.Core, etc specified to be sure that they aren't also changing?